sourcerer-dbstore/src/main/java/org/elder/sourcerer2/dbstore/DbstoreUnexpectedVersionException.kt

```diff
@@ -0,0 +1,13 @@
+package org.elder.sourcerer2.dbstore
+
+sealed class DbstoreUnexpectedVersionException : Exception()
+
+class NotFoundWhenExpectedException() : DbstoreUnexpectedVersionException()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Empty block of code detected. As they serve no purpose they should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.elder.org/coding_rules#rule_key=detekt-kotlin%3AEmptyDefaultConstructor)